src/cc/usdt/usdt.cc

```diff
@@ -36,15 +36,15 @@ Location::Location(uint64_t addr, const std::string &bin_path, const char *arg_f
     : address_(addr),
       bin_path_(bin_path) {
 
-#ifdef __aarch64__
+#if defined(__aarch64__)
```

Comment:
Is it necessary to change `#ifdef A` into `#if defined(A)` ?

Reply:
Agree.  ifdef -> defined(...) change seems unnecessary.